src/dotnet/UI.Blazor/Components/ContextMenu/context-menu.ts

```diff
@@ -65,7 +65,8 @@ export class ContextMenu {
     }
 
     private updatePosition(menu: HTMLDivElement) {
-        const maxBottom = document.documentElement.clientHeight - 90
+        const editor = document.querySelector('.chat-message-editor');
```

Comment:
What is relation between chat-message-editor and context menu? Why does context menu know about chat-message-editor?